tile_map/src/image_cache.cpp

```diff
@@ -248,7 +252,7 @@ namespace tile_map
 
   void ImageCache::NetworkError(QNetworkReply::NetworkError error)
   {
-    ROS_ERROR("NETWORK ERROR: %d", error);
+    RCLCPP_ERROR(logger_, "NETWORK ERROR: %d", error);
     // TODO add failure
```

Comment:
Again, noticed an old TODO in passing.

Reply:
I'll clarify what this TODO is about, although I'm not going to finish this quite yet (and it should also be done for the ROS1 branch)